mlir/test/Dialect/Affine/control-flow-sink.mlir

```diff
@@ -0,0 +1,85 @@
+// RUN: mlir-opt -split-input-file -control-flow-sink %s | FileCheck %s
+
+#set = affine_set<(d0) : (-d0 + 3 >= 0)>
+#map = affine_map<(d0) -> (d0)>
+
+func.func @test_affine_if_sink(%arg1: tensor<4xf32>) -> tensor<4xf32> {
+  %0 = tensor.empty() : tensor<4xf32>
+  %1 = linalg.generic {indexing_maps = [#map, #map], iterator_types = ["parallel"]} ins(%arg1: tensor<4xf32>) outs(%0: tensor<4xf32>) {
+  ^bb0(%in: f32, %out: f32):
```

Comment:
Nit: Indent generic body everywhere, maybe ins/outs could be on a separate line as well

Reply:
Hm, just double checked and I'm using the same indentation that people use elsewhere. I moved the ins/outs to a different line.